bitnami/metrics-server/templates/deployment.yaml

```diff
@@ -7,6 +7,9 @@ spec:
   replicas: {{ .Values.replicas }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+  {{- if .Values.updateStrategy }}
+  strategy: {{ toYaml .Values.updateStrategy | nindent 4 }}
```

Comment:
Please use the common.tplvalue function

Reply:
Updated by https://github.com/bitnami/charts/pull/6917/commits/464e25525586741f3d3e6fb2cc9401a2fa7a27c9